testimonials/README.md

```diff
@@ -0,0 +1,65 @@
+# Haskell.org Testimonials
+
+Is your company using haskell? Do you want to tell the world about it? This is
```

Comment:
Should it be Haskell rather than haskell here and couple other places?

Reply:
Yeah, good call, it probably should. I'll update it. Thanks!